index.html

```diff
@@ -1148,13 +1150,14 @@ <h4>Proof Configuration (ecdsa-jcs-2019)</h4>
             </li>
             <li>
 If |proofConfig|.|type| is not set to `DataIntegrityProof` and/or
-|proofConfig|.|cryptosuite| is not set to `ecdsa-jcs-2019`, an
-`INVALID_PROOF_CONFIGURATION` error MUST be raised.
+|proofConfig|.|cryptosuite| is not set to `ecdsa-jcs-2019`,
+an error MUST be raised and SHOULD convey an error type of
+`PROOF_GENERATION_ERROR`.
             </li>
             <li>
 If |proofConfig|.|created| is set and if the value is not a
-valid [[XMLSCHEMA11-2]] datetime, an `INVALID_PROOF_DATETIME` error MUST be
-raised.
+valid [[XMLSCHEMA11-2]] datetime, an an error MUST be raised and SHOULD convey
```

Comment:
```suggestion
valid [[XMLSCHEMA11-2]] datetime, an error MUST be raised and SHOULD convey
```

Reply:
I got some weird GitHub errors when trying to batch in all the changes. So had to do it in two batches.